test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -150,15 +152,18 @@ func TestSchedule1000Node30KPods(t *testing.T) {
 // 	}
 // }
 
+// testConfiguration contains the some input parameters needed for running test-suite
 type testConfig struct {
-	numPods                   int
-	numNodes                  int
+	// Note: We don't need numPods, numNodes anymore in this struct but keeping them for backward compatibility
+	numPods  		  int
+	numNodes 		  int
 	nodePreparer              testutils.TestNodePreparer
 	podCreator                *testutils.TestPodCreator
 	schedulerSupportFunctions scheduler.Configurator
 	destroyFunc               func()
 }
 
+// baseConfig returns a basic version of schedulerConfig.
 func baseConfig() *testConfig {
```

Comment:
`// baseConfig returns a minimal testConfig to be customized be different tests.`